direct/tpm2/certify_test.go

```diff
@@ -160,3 +160,100 @@ func TestCertify(t *testing.T) {
 		t.Errorf("Attested buffer is different from original buffer")
 	}
 }
+
+func TestCreateAndCertifyCreation(t *testing.T) {
+	thetpm, err := simulator.OpenSimulator()
+	if err != nil {
+		t.Fatalf("could not connect to TPM simulator: %v", err)
+	}
+	defer thetpm.Close()
+
+	public := tpm2b.Public{
+		PublicArea: tpmt.Public{
+			Type:    tpm.AlgRSA,
+			NameAlg: tpm.AlgSHA256,
+			ObjectAttributes: tpma.Object{
```

Comment:
at some point, we should have similar helper tpma.Object structs that legacy go-tpm has, like `FlagSealDefault` and `FlagSignerDefault` (https://github.com/google/go-tpm/blob/master/tpm2/constants.go#L256-L260). Writing this out can be a bit tedious

Reply:
Agreed. I think that should be made in a separate PR though.